### PR TITLE
release changelog for 2.32

### PR DESCRIPTION
**Pull request by kyrofa**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,39 @@
+snapcraft (2.32) xenial; urgency=medium
+
+  [ Michał Sawicz ]
+  * autotools: don't force uniqueness on configflags (#1337)
```
Comment (elopio) [inline debian/changelog]: autotools plugin, to be consistent with the others.
I need to get better at catching these before merge.

Comment (kyrofa) [inline debian/changelog]: Whoops, yes thank you.

Comment (kyrofa) [inline debian/changelog]: Fixed.

**Comment by kyrofa** [comment]
Looks like we have some pyramid problems:

```
```
======================================================================
ERROR: test_waf_plugin (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_waf_plugin
Traceback (most recent call last):
  File "/usr/lib/python3.5/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/lib/python3.5/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/tmp/autopkgtest.drQhAp/build.V7s/snapcraft/integration_tests/plugins/test_waf_plugin.py", line 17, in <module>
    import integration_tests
  File "/tmp/autopkgtest.drQhAp/build.V7s/snapcraft/integration_tests/__init__.py", line 38, in <module>
    from snapcraft.tests import (
  File "/tmp/autopkgtest.drQhAp/build.V7s/snapcraft/snapcraft/tests/__init__.py", line 30, in <module>
    from snapcraft.tests import fake_servers, fixture_setup
  File "/tmp/autopkgtest.drQhAp/build.V7s/snapcraft/snapcraft/tests/fixture_setup.py", line 33, in <module>
    from snapcraft.tests.fake_servers import (
  File "/tmp/autopkgtest.drQhAp/build.V7s/snapcraft/snapcraft/tests/fake_servers/api.py", line 23, in <module>
    from pyramid import response
ImportError: No module named 'pyramid'
```
```

**Comment by elopio** [comment]
With xenial-amd64, xenial-armhf, yakkety-amd64, yakkety-arm64, zesty-amd64 and zesty-arm64 I'm pretty confident that there's no regression on the deb (even though we will just tag this to release the snap). The remaining errors are clearly connection issues, and one time out from a slow running machine. Artful machines seem just borked. I will keep an eye on the beta executions during the following days, but this revision is good to beta. After manual testing during the weekend, we can move it to candidate.

**Pull request merged**